tensorboard/plugins/image/summary_v2.py

```diff
@@ -27,6 +27,41 @@
 def image(name, data, step=None, max_outputs=3, description=None):
     """Write an image summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
+
+    Logs a set of images to the current log directory for later analysis
+    in TensorBoard's 'Images' dashboard. Like `tf.summary.scalar` points,
+    each set of images is associated with a `step` and a `name`.  All the
+    image sets with the same `name` constitute a time series of image sets.
+
+    This example writes 2 random grayscale images:
+
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/logs')
+    with test_summary_writer.as_default():
+      image1 = tf.random.normal(shape=[8, 8])
```

Comment:
I'd probably use [`uniform`](https://www.tensorflow.org/api_docs/python/tf/random/uniform) instead so that it emits data that's within the valid range, since `normal` will be evenly distributed around 0.0 so half the image will get clipped.

Reply:
Done, good idea.